src/listener/listener-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "broker_urls": "10.80.66.151:9092,10.80.66.151:7092,10.80.66.151:8092",
+     "zk_url": "10.80.66.151:2181",
+     "topic": "client-b",
+     "peerOrg":"org1",
+     "peerEventUrl":"grpcs://localhost:7053",

```

Comment:
'grpcs' should be 'grpc'?

Reply:
I ran the experiment against tls enabled network. Hence grpcs.

Reply:
got it.